nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -188,8 +190,10 @@ def _execute_op(
         if is_debug() and node is not None:
             ctx.register_node_call(node)
 
-    result = trace_tensors(result, node, ctx)
-    result = ctx.execute_post_hooks(op_address, result)
+    unwrapped_result = maybe_unwrap_from_torch_return_type(result)
```

Comment:
It is necessary for training in case a post hook after a max/min/sort and etc. is present in the model. The cost of the wrapping for our target scope is two `isinstance` calls for each operation in graph as there was no post hook after a max/min/sort and etc op yet